payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -83,7 +79,6 @@ internal class PaymentSheetViewModel @Inject internal constructor(
     private val paymentMethodsRepository: PaymentMethodsRepository,
     private val paymentFlowResultProcessorProvider:
         Provider<PaymentFlowResultProcessor<out StripeIntent, StripeIntentResult<StripeIntent>>>,
-    private val googlePayRepository: GooglePayRepository,
```

Comment:
since `GooglePayRepository` is not needed, we don't need to inject it, please remove `PaymentSheetViewModelModule.provideGooglePayRepository`

Reply:
Removing on top of https://github.com/stripe/stripe-android/pull/4077.